### PR TITLE
You will now cool down when putting you get put out

### PR DESCRIPTION
**Pull request by TiviPlus**
## About The Pull Request

oneline change

## Why It's Good For The Game

Aghaghaghagh spongebob me bob the pyros retarded and flamed me again! agagaga I'm fucking dying to overheating aghhhhhhhh *deathgasp

## Changelog
:cl:
balance: If you get put out you will also slightly cool down
/:cl:

### CODE REVIEW

**Comment by jroinc** [comment]
+1, we probably shoulda had this a while ago.

File: code/modules/mob/living/living_defense.dm [hunk 1]
```diff
@@ -154,6 +154,7 @@
 	if(!on_fire)
 		return FALSE
 	on_fire = FALSE
+	adjust_bodytemperature(bodytemperature - 80, 80)
```
Comment (ma44) [inline code/modules/mob/living/living_defense.dm:157]: ``min_temp`` is set to 80 via parameter, making it possible to freeze a marine, is that intended?

Comment (TiviPlus) [reply code/modules/mob/living/living_defense.dm:157]: The lower bound shouldnt matter, in theory, but I'll set it to 300 anyway

File: code/modules/mob/living/living_defense.dm [hunk 2]
```diff
@@ -154,6 +154,7 @@
 	if(!on_fire)
 		return FALSE
 	on_fire = FALSE
+	adjust_bodytemperature(bodytemperature - 80, 300)
```
Comment (psykzz) [inline code/modules/mob/living/living_defense.dm:157]:
```suggestion
	adjust_bodytemperature(-80, 300)
```

**Pull request merged**